solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/util/SolaceDeliveryCountException.java

```diff
@@ -0,0 +1,8 @@
+package com.solace.spring.cloud.stream.binder.util;
+
+public class SolaceDeliveryCountException extends RuntimeException {
+
+    public SolaceDeliveryCountException(Throwable throwable) {
+        super(throwable);
+    }
+}
```

Comment:
don't actually need this exception. The `UnsupportedOperationException` is a `RuntimeException`, so it can be caught directly by the `XMLMessageMapper` add runtime.